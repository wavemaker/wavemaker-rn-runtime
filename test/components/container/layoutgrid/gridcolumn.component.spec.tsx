import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmGridcolumn from '@wavemaker/rn-runtime/components/container/layoutgrid/gridcolumn/gridcolumn.component';

describe('Test Gridcolumn component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmGridcolumn name="test_Gridcolumn"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});