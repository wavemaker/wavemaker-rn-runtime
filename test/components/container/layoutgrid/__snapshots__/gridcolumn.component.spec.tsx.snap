// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Gridcolumn component Check validity of sample component 1`] = `
<View
  style={
    [
      {
        "width": "100%",
      },
      undefined,
      {},
    ]
  }
/>
`;
